__tests__/core/type-system.test.ts

```diff
@@ -1172,3 +1169,21 @@ test("maybe / optional type inference verification", () => {
     }
   )
 })
+
+test("object creation with no props", () => {
+  const MyType = types.model().views((_self) => ({
+    get test() {
+      return 5
+    }
+  }))
+
+  MyType.create()
+  MyType.create({})
+  MyType.create({ [Symbol("test")]: 5 })
```

Comment:
**suggestion:** @thegedge - this is a little unintuitive to me. What do you think about dropping a comment here about why this works (it would also help me understand, myself lol)

Reply:
Actually, I should have prompted a question here. My bad!

Would we like this to also fail? I can definitely put in the work to make it so, but curious what you think? I do recall seeing an issue about supporting symbol keys for props, so I could make this fail in anticipation of that.

Reply:
I think we should make this fail. I just realized this fails for the case where an object does have props, so we should stay consistent!